src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task{
```

Comment:
Missing whitespace after Task
```suggestion
public class Event extends Task {
```